src/features/workflow/state/workflowState.ts

```diff
@@ -69,6 +77,7 @@ export interface WorkflowState {
   // with working state to determine isDirty
   workflowBase: WorkflowBase
   isDirty: boolean
+  lastDryRunID?: string,
```

Comment:
why do we need to keep try of both the last dry run and the last run? once one or the other happens, wouldn't they wipe out any previous ui anyway? probably missing something, but want to flag it!

Reply:
One is used in the Run log to determine whether the Run now should be disabled, the other is used in the workfloweditor.  Assuming two runs can be happening simultaneously, we don't want them messing with each other's UI